proto/v1/info.proto

```diff
@@ -60,4 +60,19 @@ message VariableGroupInfoRequest {
   // only contain the child variable (group) if it has data for at least one
   // of the entities.
   repeated string constrained_entities = 2;
+}
+
+
+message BulkVariableGroupInfoRequest {
+  // The variable group dcids to query for.
+  // If not specified, then return all the variable group info.
```

Comment:
nit: If we're using "(optional)" convention, use for this field too.

Reply:
Done